src/pages/receipt.js

```diff
@@ -7,9 +7,21 @@ import ImpactAlternatives from '../components/receipt-view/receipt-section/Impac
 import DataSection from '../components/receipt-view/data-section/DataSection';
 import SEO from '../components/seo';
 
-const ReceiptPage = ({ cartState }) => (
+const ReceiptPage = ({ cartState }) => {
+  const checkItems = Object.keys(cartState).map((itemKey) => {
+    if (cartState[itemKey] > 0) {
+      return true;
+    }
+    return null;
+  }).filter((item) => item);
```

Comment:
I believe you could also achieve this with 
`const cartHasItems = !Object.keys(cartState).every((item) => !cartState[item])`
`if(!cartHasItems && typeof window !== 'undefined')`